config/examples/dot_product.yaml

```diff
@@ -0,0 +1,62 @@
+---
+type: config
+log_level: DEBUG
+cpu_affinity: [0,1,2,3]
+num_local_freq: 1
+samples_per_data_set: 4608
+
+# Constants
+sizeof_float: 4
+
+cpu_monitor:
+    enabled: true 
+    track_length: 2 # save last 2 mins cpu usage.
+
+# Create pool of metadata for each buffer
+main_pool:
+  kotekan_metadata_pool: chimeMetadata
+  num_metadata_objects: 30
```

Comment:
If we want to include the concept of metadata in this tutorial, then it should be done with an `ExampleMetadata` type not the CHIME Metadata, and we should do something with it in the example stages.  For simplicity, I think we can just remove it entirely for now.  Along with the lines `metadata_pool: main_pool` below.  The buffer will default to no metadata if a pool isn't given.